### ISSUE TITLE
Edits to tech doc

### ISSUE THREAD

**Issue by RDmitchell** [LBNL-ETA/AERCalc#15]
* Holiday definition -- weather files don’t have holidays, so we’re not modeling holidays
* Start dates in table 20 -- where did these values come from? Charlie to look into this.
* Houston is missing from Table 20

**Comment by RDmitchell** [comment]
Start Dates in Table 20

Email thread -- oldest is at the bottom, latest is at the top
*******
Reply from Robin Tue, Jun 30, 2020 at 4:56 PM

**So do we need to change the technical doc and AERCalc to line these dates up with the weather file stat data?** 

Seems like we might? 

*******
Reply from Charlie Tue, Jun 30, 2020 at 4:39 PM

I would think that it should align with the .stat file, so it must have been an error.

*******
Reply from Robert Tue, Jun 30, 2020 at 2:54 PM

It appears as though the start of heating and cooling seasons are picked at a date halfway between the autumn and spring seasons, so I think that is our answer.

I did a quick check of the Minn stat file and the months don't align with the table though, so we either had an error in pulling the info for these tables or the info was taken from another source...

*******
Email from Charlie Tue, Jun 30, 2020 at 1:40 PM

The assumptions are listed in Attachment Parametrics_clean.docx, but this document does not have definition of seasons. There is spreadsheet file Heating and Cooling Seasons.xlsx, which just includes tables. I looked at E+ weather data files and found that seasons are defined in .stat file. I am attaching excerpt from the file for Chicago. The other two files are also attached. 

BTW, I am creating root level folder Attachment Energy Analysis and will extract files there. There is about 150,000 files, so they do not exceed the limit.

Charlie

- Summer is Jun:Aug
     Extreme Summer Week (nearest maximum temperature for summer)
     Extreme Hot Week Period selected: Jul 13:Jul 19, Maximum Temp=  35.00°C, Deviation=| 8.451|°C
     Typical Summer Week (nearest average temperature for summer)
     Typical Week Period selected: Aug 24:Aug 30, Average Temp=  22.35°C, Deviation=| 0.095|°C

 - Winter is Dec:Feb
     Extreme Winter Week (nearest minimum temperature for winter)
     Extreme Cold Week Period selected: Jan 27:Feb  2, Minimum Temp= -22.80°C, Deviation=|12.651|°C
     Typical Winter Week (nearest average temperature for winter)
     Typical Week Period selected: Dec 22:Dec 28, Average Temp=  -3.65°C, Deviation=| 0.123|°C

 - Autumn is Sep:Nov
     Typical Autumn Week (nearest average temperature for autumn)
     Typical Week Period selected: Oct 27:Nov  2, Average Temp=  11.28°C, Deviation=| 0.020|°C

 - Spring is Mar:May
     Typical Spring Week (nearest average temperature for spring)
     Typical Week Period selected: Apr 26:May  2, Average Temp=   9.69°C, Deviation=| 0.357|°C
